contrib/labstack/echo.v4/echotrace.go

```diff
@@ -13,18 +13,20 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpinstr"
 
 	"github.com/labstack/echo/v4"
 )
 
 // Middleware returns echo middleware which will trace incoming requests.
 func Middleware(opts ...Option) echo.MiddlewareFunc {
+	cfg := new(config)
+	defaults(cfg)
+	for _, fn := range opts {
+		fn(cfg)
+	}
+
```

Comment:
Unrelated change but I though it would be ok to do it here.
Please let me know if you prefer a separate PR.

Reply:
Fine by me, but can we please remove the empty line? nitty nit

Reply:
Fixed